source/renderer/util/template.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Please rename the file to 'render-template.js', please feel free to also add unit-tests.

Reply:
👍 I'll look into that.

Reply:
I've added some tests.

Have you considered using Jest? https://jestjs.io/

Reply:
Thanks for the test! Concerning jest: No, not yet, but we can certainly discuss the benefits over mocha in a separate issue!